cmd/hepa/main.go

```diff
@@ -235,6 +237,8 @@ var runCmd = &cli.Command{
 
 		// prometheus HTTP endpoint: /metrics
 		go func() {
+			runtime.SetBlockProfileRate(10)
+			runtime.SetMutexProfileFraction(10)
```

Comment:
If these knock things over in prod:

- try increasing the numbers by an order of mag
- ...or just comment these back out entirely.